### ISSUE TITLE
A request for dataset of paper "ST-MTL: Spatio-Temporal Multitask Learning Model to Predict Scanpath While Tracking Instruments in Robotic Surgery"

### ISSUE THREAD

**Issue by Yuqi99** [Vibashan/irg-sfda#6]
Dear Dr. Vibashan,

I am a graduate student from Changchun University of Science and Technology in China, focusing on machine learning and saliency prediction.

Recently, I have had the opportunity to read your paper, "ST-MTL: Spatio-Temporal Multitask Learning Model to Predict Scanpath While Tracking Instruments in Robotic Surgery". I find the research intriguing and have been studying it in detail. I am particularly interested in the prediction of saliency maps in surgical scenarios.

However, the training sets 1, 2, 3, 5, 6, and 8 are not publicly available. Additionally, the dataset from the 2017 MICCAI Surgical Instrument Segmentation Challenge only contains ground truth segmentation masks and lacks ground truth saliency maps.

I would be immensely grateful if you could share the complete dataset and any relevant information that may be essential for my research. I assure you that the dataset will be utilized solely for academic purposes.

I am deeply appreciative of your consideration and look forward to your response at your earliest convenience.

Sincerely,
Yuqi Yan

**Comment by Vibashan** [comment]
Hi @Yuqi99 ,

The saliency maps are generated as explained in ([https://link.springer.com/chapter/10.1007/978-3-030-32254-0_46](https://link.springer.com/chapter/10.1007/978-3-030-32254-0_46)) and we followed it in our paper.

I think you can generate the dataset by writing a simple Python script following the paper.
